jsm.go

```diff
@@ -268,6 +285,30 @@ type consumerResponse struct {
 
 // AddConsumer will add a JetStream consumer.
 func (js *js) AddConsumer(stream string, cfg *ConsumerConfig, opts ...JSOpt) (*ConsumerInfo, error) {
+	if cfg != nil && cfg.Durable != _EMPTY_ {
+		consInfo, err := js.ConsumerInfo(stream, cfg.Durable)
```

Comment:
Yeah, I would lean towards leaving that, as without that `AddConsumer()` I feel does not work as the name suggests (it will simply update a consumer if it already exists).